code/API_definitions/connected-network-type-subscriptions.yaml

```diff
@@ -44,6 +44,17 @@ info:
       - the Access Token `sinkCredential` (optionally set by the requester) expiration time has been reached
       - the API server has to stop sending notification prematurely
 
+    **Note on combined usage of ``initialEvent`` and ``subscriptionMaxEvents``**:
+
+    If an event is triggered following ``initialEvent`` set to true,
+    this event will be counted towards ``subscriptionMaxEvents`` (if provided).
+
+
+    **Clarification on ``initialEvent`` & ``event-type`` behaviour:**
+
+    When ``initialEvent`` is set to ``true`` in the subscription request, a notification has to be sent immediatly to provide the connected network type of the device targeted.
+    If the device is not connected, `UNKNOWN` is sent as ``connectedNetworkType`` in this notification.
+

```

Comment:
- typo in "immediatly"
- UNKNOWN is sent if connection [technology] can not be determined.

Reply:
For the second point @akoshunyadi. If you subscribe with initalEvent set to true and the device si not connected (suppose the device is shut off) then we did trigger any event then?

I'm fine with that but looking for feedback.
+ @eric-murray  & @sachinvodafone  to get their view.

Reply:
I wanted to point out the UNKNOWN is defined as "can not be determined" and not "not connected". Not connected is one of the cases when it can't determined, but there are other use cases for that too.
To your question: I think we should send the initial event, even if the status is unknown.

Reply:
OK got it. 
If the status can not be determined we send an inital event valued to UNKNOWN. If the mobile is not connected no event triggered. right? I update